core/components/atoms/metricInput/__tests__/MetricInput.test.tsx

```diff
@@ -241,3 +241,16 @@ describe('MetricInput component with props showActionButton', () => {
     expect(queryByTestId('DesignSystem-MetricInput--downIcon')).not.toBeInTheDocument();
   });
 });
+
+describe('MetricInput component receives focus', () => {
+  it('when MetricInputWrapper is clicked', () => {
+    const { getByTestId } = render(<MetricInput />);
+    const metricInputWrapper = getByTestId('DesignSystem-MetricInputWrapper');
+    const metricInput = getByTestId('DesignSystem-MetricInput');
+
+    metricInput.focus = FunctionValue;
```

Comment:
Please verify why we are storing the function value in `metricInput.focus` variable?

Reply:
Beacuse we need to verify whether this function is being called or not. Hence using a mock for the same.